_data/events1.yml

```diff
@@ -0,0 +1,78 @@
+- category: Online Event
+  category_name: Online Event
+  date: Tue, Aug 13, 2024
+  description: "Kedro is an open-source Python framework to create reproducible, maintainable,\
+    \ and modular data science code. It uses software engineering best practices to\
+    \ help you build production-ready data science pipelines.\nSchedule:\n\u23F0 19:00\
+    \ - 19:15 Intro to Women Coding Community\n\u23F0 19:15 - 19:45: Talk by Merel\
+    \ Theisen\n\u23F0 19:45 - 20:00: Q&A"
+  expiration: '20240813'
+  host: Silke Nodwell and Irina Kamalova
+  image:
+    alt: 'Kedro: An open-source project for building maintainable data science pipelines
+      cover photo'
+    path: https://secure.meetupstatic.com/photos/event/6/4/c/e/event_522085806.webp

```

Comment:
Coud you upload the photo? into `assets/images/events/event_522085806.webp` and keep here just the path?